src/routes.js

```diff
@@ -39,25 +38,22 @@ const getRoutes = (vms) => ([
       },
       {
         path: '/vm/:id',
-        title: (match) => {
-          return vms.getIn(['vms', match.params.id, 'name'])
-        },
+        title: (match) => { return vms.getIn(['vms', match.params.id, 'name']) },
```

Comment:
What was bad about the previous formatting? Or how about just `(match) => vms.getIn(['vms', match.params.id, 'name'])`?

Reply:
It was just inconsistent and I find the single line function easier to scan quickly.  Taking the extra block out looks even better.  I'll do that.